src/ensembl/src/genome/genomeActions.ts

```diff
@@ -21,12 +28,30 @@ export const fetchGenomeInfoAsyncActions = createAsyncAction(
 
 export const fetchGenomeData: ActionCreator<
   ThunkAction<void, any, null, Action<string>>
-> = (genomeId: string) => async (dispatch) => {
+> = (genomeId: string) => async (dispatch, getState: () => RootState) => {
   await Promise.all([
     dispatch(fetchGenomeInfo(genomeId)),
     dispatch(fetchGenomeTrackCategories(genomeId))
   ]);
 
+  const state = getState();
+  const committedSpecies: CommittedItem[] = getEnabledCommittedSpecies(state);
+  const genomeInfo: GenomeInfo | null = getGenomeInfoById(state, genomeId);
```

Comment:
Don't we automatically get types for committedSpecies and genomeInfo from selectors?

Reply:
It's jus that there was no indication to say that it's not necessary. 
Removed it now.